extensions/cornerstone/src/init.js

```diff
@@ -119,28 +183,25 @@ export default function init({ servicesManager, configuration }) {
           title: 'Enter your annotation',
           value: { label: data ? data.text : '' },
           noCloseButton: true,
-          onClose: () => UIDialogService.dismiss({ id: dialogId }),
+          onClose: () => UIDialogService.dismiss({ id: 'enter-annotation' }),
           actions: [
             { id: 'cancel', text: 'Cancel', type: 'secondary' },
             { id: 'save', text: 'Save', type: 'primary' },
           ],
-          onSubmit: ({ action, value }) => {
-            switch (action.id) {
-              case 'save':
-                callback(value.label);
-            }
-            UIDialogService.dismiss({ id: dialogId });
-          },
+          onSubmit: ({ action, value }) => onSubmitHandler({ action, value }),
```

Comment:
Can this just be onSubmit: onSubmitHandler ?

Reply:
Updated, thanks.